examples/tohoku_inversion/forward_run.py

```diff
@@ -0,0 +1,29 @@
+from thetis import *
+import time as time_mod
+from model_config import *
+import argparse
+import os
+
+
+# Parse user input
+parser = argparse.ArgumentParser(
+    description="Tohoku tsunami propagation",
+    formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+)
+parser.add_argument("--source_model", type=str, default="CG1")
+args = parser.parse_args()
+source_model = args.source_model
+
+# Solve forward
+pwd = os.path.abspath(os.path.dirname(__file__))
```

Comment:
This is needed for the tests to work? Ideally tests should not write to disk. Or if needed we should use a temp directory created with pytest setup.